src/pages/components/[libId]/[componentId].tsx

```diff
@@ -62,10 +64,30 @@ export const ComponentPage = ({
         return null;
     }
 
+    const sections = useMemo<Section[]>(() => {
+        return libs.map((lib) => ({
+            id: lib.id,
+            title: lib.title,
+            // url: `/components/${lib.id}`, // "Overview" link
+            subSections: lib.components.map((component) => ({
```

Comment:
Let's fix eslint warnings:

<img width="877" alt="image" src="https://github.com/gravity-ui/landing/assets/29424746/ab6cd912-dddf-44bc-9cef-f2481736d2c7">

Reply:
<img width="808" alt="image" src="https://github.com/gravity-ui/landing/assets/42549523/c4d0f7d6-4ffd-433d-b01b-a1682dd67025">
I have no warnings

Reply:
Looks like a problem with vscode eslint plugin.
Try to run `npm run lint`.

<img width="1474" alt="image" src="https://github.com/gravity-ui/landing/assets/29424746/e39ecc0b-5dec-4fa9-8549-0543f1130415">